spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDao.java

```diff
@@ -79,6 +85,12 @@
 	private static final String FIND_TASK_EXECUTIONS = "SELECT TASK_EXECUTION_ID, "
 			+ "START_TIME, END_TIME, TASK_NAME, EXIT_CODE, "
 			+ "EXIT_MESSAGE, LAST_UPDATED, STATUS_CODE "
+			+ "from %PREFIX%EXECUTION order by TASK_EXECUTION_ID "
```

Comment:
This will need to be updated to be a sequence number (spring-cloud/spring-cloud-task#34) to truly be useful.  Sorting by a random GUID will be just junk.  At least we should sort by start time for the time being?